ui/de-lib/src/main/java/org/iplantc/de/client/util/OntologyUtil.java

```diff
@@ -41,6 +41,11 @@ public int compare(OntologyHierarchy o1, OntologyHierarchy o2) {
     final String UNCLASSIFIED_LABEL = "Unclassified";
     final String UNCLASSIFIED_IRI_APPEND = "_unclassified";
 
+    final String BETA_ATTR = "n2t.net/ark:/99152/h1459";
+    final String BETA_VALUE = "beta";
+    final String BETA_SUB_VALUE = "releaseStatus";
+    final String BETA_SUB_UNIT = "attr";
```

Comment:
These values should be probably be stored as config values in the UI's properties file, rather than hard-coded.
In fact, these values are already duplicated in the apps service configs, so they could be refactored into the group_vars for use in both properties file templates.